docs/releases.rst

```diff
@@ -391,4 +518,4 @@ This release adds support for *cascading replication* and simplifies Patroni man
 
   The tests can be launched manually using the *behave* command. They are also launched automatically for pull requests and after commits.
 
-  Releases notes for some older versions can be found on `project's github page <https://github.com/zalando/patroni/releases>`__.
\ No newline at end of file
+  Releases notes for some older versions can be found on `project's github page <https://github.com/zalando/patroni/releases>`__.
```

Comment:
Not sure what changed on this line.  Newline removed or added?  In any case, there's a pre-existing typo: "Release notes".